src/components/notebook/notebook.scss

```diff
@@ -16,47 +16,62 @@
   }
 
   .notebook-tab {
+    position: relative;
+    bottom: -1px;
     display: flex;
     align-items: center;
     box-sizing: border-box;
     height: 40px;
     border-radius: 10px 10px 0 0;
-    padding: 0 10px;
     border: solid 1px $gray-dark;
     background-color: $gray-dark10;
-    width: fit-content;
+    padding: 4px 4px 0 4px;
     cursor: pointer;
-    position: relative;
-    bottom: -1px;
+
+    .inner {
+      display: flex;
+      align-items: center;
+      border-radius: 5px 5px 0 0;
+      padding: 0 7px;
+      width: fit-content;
+      height: 36px;
+    }
 
     .icon {
-      margin: 5px;
+      margin-right: 5px;
+    }
+
+    &.react-tabs__tab--selected {
+      background-color: white;
+      border-bottom: solid 1px white;
+      pointer-events: none;
```

Comment:
Cursor is still a pointer even though tab has been selected. Second click puts a blue outline on the tab label
![image](https://user-images.githubusercontent.com/7716653/107318370-7901fa00-6a51-11eb-9e5b-2fb67f9dfe0c.png)

Reply:
I added an explicit switch to `cursor: auto` - but as with all of these React state switches it does require the mouse to move at least one pixel if you stay hovered over it.  The selection state we should leave alone until we can make a broader accessibility pass.